js/templates/userPage/store.html

```diff
@@ -68,6 +68,12 @@
           </div>
         </div>
       </div>
+      <% if (!ob.vendor) { %>
+        <div class="contentBox row pad clrP clrBr">
```

Comment:
Piggy backing on the comment below... maybe changing this to:

`<div class="contentBox row pad clrP clrBr <% if (ob.vendor) print('hide')%>">` and then toggling that hide class on `change:vendor`?